OA-PASS/pass-docker
# Add schema service and update to context 3.3

* Adds the schema service, proxied to `https://pass.local/schemaservice`
* Pre-loads context 3.3 in Fedora
* Updates assets image to add `schemas` properties pointing to the proper schemas

# To test
* Bring everything up (`docker-compose pull && docker-compose up -d`).  
* On the command line, you can use `curl` to make a request through the back door (unencrypted).  The following query requests the schema service to generate schemas for all repositories.  If you look at the results, it has common, nihms, and jscholarship in that order.
  <pre>
    curl -X POST --data '["https://pass.local/fcrepo/rest/repositories/41/96/0a/92/41960a92-d3f8-4616-86a6-9e9cadc1a269",
     "https://pass.local/fcrepo/rest/repositories/9c/10/6a/13/9c106a13-122b-4775-8c04-4f9c926ee451",
     "https://pass.local/fcrepo/rest/repositories/77/cc/80/64/77cc8064-a918-4823-968d-2b17386db76d",
     "https://pass.local/fcrepo/rest/repositories/77/64/12/ec/776412ec-0f5e-488e-97dc-15bb427d27e2"]'  http://localhost:8086 </pre>
* To test that the shib proxy is actually proxying requests to the schema service, point your browser to `https://pass.local/schemaservice`.  You should see a default splash page.  Feel free to POST to this endpoint as well.  If using curl, realize that it needs the shib cookie as part of this request.
 
Resolves OA-PASS/metadata-schemas#8
